packages/custom-webpack/src/custom-webpack-builder.ts

```diff
@@ -67,7 +67,10 @@ function resolveCustomWebpackConfig(path: string): CustomWebpackConfig {
   // `module.exports = { ... }`. And the second one is:
   // `export default { ... }`. The ESM format is compiled into:
   // `{ default: { ... } }`
-  if (typeof customWebpackConfig.default === 'object') {
+  if (
```

Comment:
What do you think about this?
```js
return customWebpackConfig.default || customWebpackConfig;
```
Less code I guess but as you wish.

Reply:
Done